docker/Dockerfile

```diff
@@ -20,12 +22,16 @@
 #   `aarch64-unknown-linux-gnu`.
 
 # Stage 1 - Generate recipe file for dependencies
-FROM lukemathwalker/cargo-chef:latest-rust-slim AS chef
+
 ARG environment=k8s-local
 # Modern Docker provides TARGETPLATFORM and friends, but these do not
 # provide enough information to reconstruct the whole Rust target and
 # switching on them to map to a subset of targets seems unwise
 ARG target=x86_64-unknown-linux-gnu
+ARG binaries=
+ARG copy=${binaries:+_copy}
```

Comment:
What does this do? Never seen this syntax

Reply:
`${foo:+bar}` substitutes `bar` if `$foo` is non-empty, else the empty string. `${foo+bar}` is the same but if `$foo` is defined.

https://tldp.org/LDP/abs/html/parameter-substitution.html is a go-to resource for me when working with bash 😅 Though note that the ones in the Dockerfile are actually Dockerfile-specific substitution syntax, which is a subset of bash substitution that I haven't found a good reference for yet.